getdess.sh

```diff
@@ -127,7 +127,23 @@ install_docker () {
         dnf -y install docker-ce docker-ce-cli containerd.io;
         systemctl enable --now docker.service;
       ;;
-      *) curl -fsSL https://get.docker.com | sh;;
+      *) 
+      # Add Docker's official GPG key:
```

Comment:
*) is the fallback case, which I was relying on get.docker.com to fulfill... we shouldn't fallback to a debian only solution. Instead we should add a new case for debian above the fallback case, and restore the old fallback case. This will break for all non-debian systems that we were using the get docker script for

Reply:
Ah okay, yeah I see, will update accordingly